### DISCUSSION TITLE
A user-friendly usage tutorial/example?

### DISCUSSION THREAD

**Discussion by fralik** [microsoftgraph/msgraph-sdk-python#746] [General]
I am trying to understand how to actually use the library, but the documentation is not great. For example, consider this https://learn.microsoft.com/en-us/graph/api/driveitem-list-children?view=graph-rest-1.0&tabs=python#list-children-in-the-root-of-the-current-users-drive

![image](https://github.com/microsoftgraph/msgraph-sdk-python/assets/191278/30b817dd-6ede-4ea4-827f-74f6c521fc72)
It suggests that one knows two pieces of information: `drive-id`, `driveItem-id`. Not only it contradicts the text in the description of the example, but it also leaves me with no clues on how to obtain these IDs. Imagine, I can go on Sharepoint website and open a folder I would like to work with. What are `drive-id`, `driveItem-id`?! Where can I see their values?

**Comment by Hrrusha** [comment]
To obtain driveid you can fetch the drive metadata:

For example: 

```
drives = await graph_client.drives.get()
for drive in drives.value:
   print(drive.id)
```
